.gitignore

```diff
@@ -28,3 +28,7 @@
 *.safetensors
 *.gguf
 *.vmfb
+*.onnx
```

Comment:
I'm not sure about ignoring .onnx files and these other artifact files across the whole repo. Some test cases / suites may benefit from checking in the stable artifact files. Might want to scope those ignores to specific directories.